velero-plugins/common/backup.go

```diff
@@ -17,7 +17,21 @@ type BackupPlugin struct {
 
 // AppliesTo returns a velero.ResourceSelector that applies to everything.
```

Comment:
NIT: Comment here is outdated. Since selector is being used.. I don't think it applies to everything any longer.

Reply:
Good catch